frontend/src/metabase/query_builder/containers/QueryBuilder.tsx

```diff
@@ -94,11 +102,30 @@ const timelineProps = {
   loadingAndErrorWrapper: false,
 };
 
-const mapStateToProps = (state, props) => {
+type BookmarkListLoaderOutput = {
+  bookmarks: BookmarkType[];
+  reloadBookmarks: () => void;
+};
+
+type TimelineListLoaderOutput = {
+  timelines: Timeline[];
+  reloadTimelines: () => void;
+};
+
+type EntityListLoaderMergedProps = {
+  allLoading: boolean;
+  allLoaded: boolean;
+  allFetched: boolean;
+  allError: boolean;
```

Comment:
```suggestion
  allError: unknown;
```

Reply:
[Trust me, I'm an engineer](https://www.youtube.com/watch?v=rp8hvyjZWHs).

----

1. Apply this patch:

```patch
diff --git a/frontend/src/metabase/entities/containers/EntityListLoader.jsx b/frontend/src/metabase/entities/containers/EntityListLoader.jsx
index 10c7b54ae5..eee1a31259 100644
--- a/frontend/src/metabase/entities/containers/EntityListLoader.jsx
+++ b/frontend/src/metabase/entities/containers/EntityListLoader.jsx
@@ -277,6 +277,8 @@ const EntityListLoader = _.compose(
       entityQuery,
     });
 
+    console.log(error || (allError == null ? null : allError));
+
     return {
       list,
       entityQuery,
```

2. Go to http://localhost:3000/browse/databases/1-sample-database
3. Block network request to GET http://localhost:3000/api/database/1/schemas
4. Refresh the page
5. Be amazed due to lack of booleans in JS console

Reply:
you do tend to bring wonder and amazement into my engineering career 😄

Reply:
certainly doesn't look like a boolean 😄 
<img width="740" alt="image" src="https://github.com/user-attachments/assets/862635fa-bd5e-4c9d-a6c3-273bd1578c0f">